lib/pages/register/index.js

```diff
@@ -0,0 +1,49 @@
+const page = require('@asl/pages/lib/page');
+const form = require('@asl/pages/pages/common/routers/form');
+const schema = require('./schema');
+const existingTemplate = require('./views/existing');
+
+module.exports = settings => {
+  const app = page({
+    ...settings,
+    root: __dirname
+  });
+
+  app.use(form({
+    schema,
+    configure: (req, res, next) => {
+      if (req.existingUser) {
+        req.form.schema = {};
+        res.template = existingTemplate.default || existingTemplate;
+      }
+      next();
+    }
+  }));
+
+  app.use((req, res, next) => {
+    res.locals.static.schema = req.form.schema;
```

Comment:
This feels like a thing that might be best done in the form router.

Reply:
Ah yeah, I actually fixed it there, I wrote this before when I was using a previous version of pages - will remove